server/db/foods.js

```diff
@@ -4,6 +4,22 @@ function getFoods (db = connection) {
   return db('foods')
 }
 
+function addFood (food, db = connection) {
+  let foodID
+
```

Comment:
No new line here please :)

Reply:
Also could we actually move this function under the `getFood` function?

Reply:
done